src/mixin-props-test.js

```diff
@@ -20,3 +20,61 @@ QUnit.test("basics", function(assert) {
 	el.age = "33";
 	assert.equal(el.age, 33, "updated age");
 });
+
+try {
+	eval(`
+		QUnit.test('Class fields should be observable', (assert) => {
+			class DefineElement extends mixinDefine(Object) {
+				greetings = 'Hello'; // jshint ignore:line
+				static get props() {
+					return {};
+				}
+			}
+			const el = new DefineElement();
+
+			if (el.hasOwnProperty('greetings')) {
+				el.initialize();
+			
+				el.on('greetings', () => {
+					assert.ok('The class field is observable');
+				});
+			
+				el.greetings = 'Hola';
+			} else {
+				assert.ok('skipped');
+			}
+	});`);
+} catch (e) {
+	//console.log("doesn't work");
+}
+
+try {
+	eval(`QUnit.test('Class fields should not overwrite static props', (assert) => {
+		class DefineElement extends mixinDefine(Object) {
+			greetings = 'Hello'; // jshint ignore:line
+			static get props() {
+				return {
+					greetings: {type: String, default: 'Bonjour'}
+				};
+			}
+		}
+	
+		const el = new DefineElement();
+			el.initialize();
+		if (el.hasOwnProperty('greetings')) {
+			el.on('greetings', () => {
+				assert.ok('The class field is observable');
+			});
+	
+			el.greetings = 'Hola';
```

Comment:
After this you should also check that setting the value to a non-string causes an error.